src/Files.App/Utils/StatusCenter/StatusCenterItemProgressModel.cs

```diff
@@ -121,11 +125,22 @@ public StatusCenterItemProgressModel(IProgress<StatusCenterItemProgressModel>? p
 			EnumerationCompleted = enumerationCompleted;
 			Status = status;
 			ItemsCount = itemsCount;
-			TotalSize = totalSize;
```

Comment:
We finally don't need to calculate total-size (recursively) beforehand. that will be calculated in parallel, drawing graph.